src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/mapView.js

```diff
@@ -69,16 +70,23 @@ document.head.appendChild(styleHead);
  */
 class MapViewVisualization extends Visualization {
     static inputType = "Any"
+    mapId            = 0;
 
     onDataReceived(data) {
         while (this.dom.firstChild) {
             this.dom.removeChild(this.dom.lastChild);
         }
 
+        if (this.mapId === 0) {
+            MAP_ID     += 1;
+            this.mapId  = MAP_ID;
+        }
```

Comment:
Create utility function for that next to global variable, make section for it and add documentation. Implementation of the incremenation of the global variable should not be part fo the loigic in this visualization.